docs/documenting/lorem-ipsum.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Documentation structure and example text
+---
+
+This page shows examples of the available documentation features (such as inline icons, text box, links to related topics, and glossary entries). It also shows the documentation structure.
+
+Use this template together with the following documentation:
+
+- The integration documentation template on under [/_integrations/_integration_docs_template.md](https://github.com/home-assistant/home-assistant.io/tree/current/source/_integrations/_integration_docs_template.md).
```

Comment:
You probably need that link to have "markdown" instead of "md" to make it work after merging the parent pr.